text/0000-plan-flags.md

```diff
@@ -9,206 +9,238 @@
 # Summary
 [summary]: #summary
 
-We would like to add the ability for buildpacks to be able to specify `build` and `launch` requirements for a dependency.
```

Comment:
Seems like the purpose of this RFC has changed somewhat. We are adding new functionality to the lifecycle to allow merging of keys. With this new behavior `build`, `launch` and `versions` and just first examples of fields we would like to merge.

Reply:
Could you elaborate a little more on what you would like this to look like?